selfdrive/car/honda/values.py

```diff
@@ -382,15 +382,14 @@ class CAR:
       b'39990-TBA-C020\x00\x00',
       b'39990-TBA-C120\x00\x00',
       b'39990-TGG-A020\x00\x00',
+      b'39990-TGG,A020\x00\x00',
       b'39990-TGG-A120\x00\x00',
```

Comment:
There is no A020, but A120 for the cars that currently have modded eps support. Line 385 should be `b'39990-TGG,A120\x00\x00',`

Reply:
I have a 2020 Civic Hatchback Sport Touring and my EPS fw version is `b'39990-TGG-A020\x00\x00'`